### ISSUE TITLE
Notifications not working properly

### ISSUE THREAD

**Issue by aimilushev** [mrgames13/Feinstaub-App#4]
Currently I've set background sync to 45 minutes and foreground to 60 seconds. Notifications normally are not fired when in background, even though the threshold is excieded multiple times ( I'm using the app for about 2-3 weeks and look at the app once a day). The only time I receive notification (not 100% of the cases) is after I enter in the app refresh the data, minimize the app and some minutes after I receive 1 notification in the notifications panel, but can hear the notifications sound for about a minute, as I recieve for example 100 notifications, which bothers me a lot. Reproduced on samsung s7 android 8.
Another improvement suggestion is to use Job scheduler/ Work manager instead of alarm manager, in order to optimize battery usage and take care of the system's doze mode.

**Comment by mrgames13** [comment]
We've implemented the JobScheduler for API level 21 and higher. Besides that we added a few conditions for firing the notifications.
- JobService executed from background
- Measured value is higher than every measured value before on that day
- JobService only sends notification when exceeding the threshold when coming from under the threshold but not when the previous value was also over it

The update will be released a few days from now. We're looking forward to other improvement suggestions and your review on Google Play.
